public/src/pages/live.vue

```diff
@@ -76,7 +78,7 @@
                             <th>Flags</th>
                         </tr>
 
-                        <tr v-for="(lap, i) in showLapsCar.laps" :key="i" v-bind:class="{ 'tbl-row': true, invalid: lap.flags>0}">
+                        <tr v-for="(lap, i) in showLapsCar.laps" :key="i" v-bind:class="{ 'tbl-row': true, invalid: lap.flags>0, best: lap.lapTimeMS === showLapsCar.bestLapMS}">
                             <td>{{i+1}}</td>
```

Comment:
`lap.flags > 0` (from previous version, but should probably be fixed), space after `MS` to match space at front